app/services/WhatYouOweService.scala

```diff
@@ -79,18 +79,21 @@ class WhatYouOweService @Inject()(val financialDetailsService: FinancialDetailsS
           chargesList = getFilteredChargesList(financialDetailsModelList, isReviewAndReconciledEnabled, isFilterCodedOutPoasEnabled),
           codedOutDocumentDetail = codedOutChargeItem)
 
-        callOutstandingCharges(mtdUser.saUtr, mtdUser.incomeSources.yearOfMigration, dateService.getCurrentTaxYearEnd).map {
+        callOutstandingCharges(dateService.getCurrentTaxYearEnd).map {
```

Comment:
* if we can pass current tax year in the next fashion:
`callOutstandingCharges(dateService.getCurrentTaxYear)` ;
* I think we added getCurrentTaxYearEnd method to TaxYear type by mistake, there is no really need to have it, we might need to raise a technical debt to drop it, see other comments below (feel free to raise this story if you like)